tests/scripts/kubeadm.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash +e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+tarname=image.tar
+tarfile=${WORK_DIR}/tests/${tarname}
+
+export KUBE_VERSION=${KUBE_VERSION:-"v1.7.2"}
+
+
```

Comment:
can you make `install()` a verb here and move the logic in kubeadm-install into here.